index.html

```diff
@@ -4,52 +4,62 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Setup Mobile First</title>
+    <title>Portfolio</title>
     <link rel="stylesheet" href="style.css" />
     <script
       src="https://kit.fontawesome.com/b1fa342dbf.js"
-      crossorigin="anonymous"
-    ></script>
+      crossorigin="anonymous"></script>
   </head>
-
   <body>
     <header>
       <div class="header flex">
         <a href="#" class="brand">Faiza</a>
         <nav>
-        <i class="fa-sharp fa-solid fa-bars"></i>
+          <i class="fa-sharp fa-solid fa-bars icons"></i>
         </nav>
-        <div>
+      </div>
     </header>
     <main>
       <section class="home flex">
```

Comment:
- [Optional] Kindly reduce the spacing above your headline section. As demanded by the project requirements, you must stick to the design template as much as possible, this includes margins and padding. Kindly fix this, please.
### Your design
![image](https://user-images.githubusercontent.com/94551484/235005399-42c3b6d5-6eff-4691-b6cf-1b0455f0740b.png)
### figma
![image](https://user-images.githubusercontent.com/94551484/235005451-944b383a-ee47-4904-8541-3a57b6cbaf91.png)